Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM alpine:latest as builder
+
+ARG KAFKA_VERSION=0.11.0.0
+ARG SCALA_VERSION=2.11
+
+RUN apk update && apk add --no-cache wget tar gzip
+
+RUN wget https://archive.apache.org/dist/kafka/0.11.0.0/kafka_${SCALA_VERSION}-0.11.0.0.tgz && \
+    tar -xzf kafka_${SCALA_VERSION}-0.11.0.0.tgz && \
+    rm kafka_${SCALA_VERSION}-0.11.0.0.tgz
```

Comment:
You've defined the `KAFKA_VERSION` arg but aren't using it in the commands that follow.